corehq/apps/es/apps.py

```diff
@@ -46,3 +47,7 @@ def uses_case_sharing(case_sharing=True):
 
 def cloudcare_enabled(cloudcare_enabled):
     return filters.term('cloudcare_enabled', cloudcare_enabled)
+
+
+def app_id(app_id):
```

Comment:
Nit: I think this name is confusing - maybe `copy_of` or `copy_of_app_id`?

Reply:
I agree that this is confusing, but we actually refer to this field as `app_id` elsewhere. For example: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/dbaccessors.py#L36
the `app_id` parameter there is pulling from the `copy_of` field here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/_design/views/applications/map.js

I think this is the right name for this field, as this is the ID that is shown in the URL which I think most people would think of as the `app_id`  (as opposed to the `build_id`).